lib/manageiq/ui/classic/engine.rb

```diff
@@ -1,7 +1,16 @@
 require 'rails/all'
-require 'sass-rails'
-require 'coffee-rails'
-require 'patternfly-sass'
+
+if ENV["RAILS_ENV"] != "production" || defined?(Rake)
```

Comment:
@NickLaMuro Can you add a comment here with the same description from the PR?  Casual readers to this will not understand the rationale for these 2 conditionals.